packages/grpc-js-core/src/call-stream.ts

```diff
@@ -9,9 +9,10 @@ import {FilterStackFactory} from './filter-stack';
 import {Metadata} from './metadata';
 import {ObjectDuplex} from './object-stream';
 
-const {HTTP2_HEADER_STATUS, HTTP2_HEADER_CONTENT_TYPE, NGHTTP2_CANCEL} = http2.constants;
+const {HTTP2_HEADER_STATUS, HTTP2_HEADER_CONTENT_TYPE, NGHTTP2_CANCEL} =
+    http2.constants;
 
-export type Deadline = Date | number;
+export type Deadline = Date|number;
```

Comment:
We seem to have troubles deciding if we want spaces or not around these :D We really need a linter...

Reply:
It's possible that this is a change by the formatter. Or me accidentally reversing a change by the formatter.